src/Modules/DatePickerInput.react.js

```diff
@@ -310,46 +294,66 @@ class DatePickerInput extends Component {
         }
     }
 
-    _getAllowedDateFormats(specifiedFormat) {
-        const formats = DateUtils.getAllowedDateFormats();
-
-        if (_.indexOf(formats, specifiedFormat) < 0) {
-            formats.unshift(specifiedFormat);
-        }
-
-        return formats;
+    _setOpen(open) {
+        this.setState({ isCalendarOpen: open });
     }
 }
 
+DatePickerInput.defaultProps = {
+    className: '',
+    controls: 'dropdowns',
+    date: undefined,
+    dateFrom: undefined,
+    dateTo: undefined,
+    disabled: false,
+    disabledDays: undefined,
+    events: undefined,
+    errorMessage: '',
+    excludeDates: undefined,
+    filterDates: undefined,
+    id: undefined,
+    includeDates: undefined,
+    label: '',
+    locale: '',
```

Comment:
Is empty string a useful default for locale?  Should we default it to en-US or whatever we're falling back on?

Reply:
I like it. I changed it.